functional/FunctionalTest.java

```diff
@@ -1692,6 +1697,13 @@ public static void main(String[] args) {
       System.exit(-1);
     }
 
+    if (System.getenv("MINIO_MINT") != null) {
```

Comment:
As we discussed previously, `DATA_DIR` is too generic name ie functional test will fail in non-mint environment.  Its required to update `run.sh` like `MINIO_MINT= ./gradlew runFunctionalTest`